tools/jdbc/src/jni/duckdb_java.cpp

```diff
@@ -187,6 +191,12 @@ static string byte_array_to_string(JNIEnv *env, jbyteArray ba_j) {
 	return ret;
 }
 
+static string jstring_to_string(JNIEnv *env, jstring string_j) {
+	jobject charset = env->CallStaticObjectMethod(J_Charset, J_Charset_forName, env->NewStringUTF("UTF-8"));
```

Comment:
This charset jobject is basically a constant, isn't it? If so, I might be moved up into the JNI_OnLoad function to avoid the JNI->Java calls every time. What do you think?

Reply:
Fixed.